README.md

```diff
@@ -744,7 +744,7 @@ Most of utilities are found by native API. Others advanced functions could be ch
 
   // Native
   function isWindow(obj) {
-    return obj != null && obj === obj.window;
+    return obj !== null obj !== undefined && obj === obj.window;
```

Comment:
`obj != null` is short for `obj !== null obj !== undefined`, and the only occasion to use `==`